client/package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "axios": "^1.6.7",
```

Comment:
I think we should stick to one http client. `Fetch` should be sufficient.

Reply:
I agree fetch should be enough for now.

Reply:
axios still in updated PR, need to remove